backend/src/main/java/ee/tlu/forum/security/SecurityConfig.java

```diff
@@ -46,14 +46,31 @@ protected void configure(HttpSecurity http) throws Exception {
         http.authorizeRequests().
                 antMatchers(AUTH_WHITELIST).permitAll();
 
-        // allow only certain roles access to these endpoints
-        http.authorizeRequests().antMatchers(GET, "/api/users/**").hasAnyAuthority("ROLE_USER");
-        http.authorizeRequests().antMatchers(GET, "/api/user/*").hasAnyAuthority("ROLE_USER");
-        http.authorizeRequests().antMatchers(POST, "/api/user/save/**").hasAnyAuthority("ROLE_ADMIN");
+        // UserController
+        http.authorizeRequests().antMatchers("/api/user/save/**").hasAnyAuthority("ROLE_ADMIN");
```

Comment:
I'm guessing this is to edit user? What if it's the user editing their own profile and they're not an admin?

Reply:
That's actually the wrong endpoint and you're right. I'll push a quickfix ASAP